lib/postprocess/image.js

```diff
@@ -93,6 +99,15 @@ module.exports = async files => {
           response.headers['content-type'] || response.headers['Content-Type']
         imageExtensions = contentType.split('/')[1]
         imageBuffer = Buffer.from(response.data)
+
+        // check if the file is valid
+        try {
+          // sizeOf also checks for the right type and validity of the buffer
+          sizeOf(imageBuffer)

```

Comment:
Performance: i think we can re-use the return value of this function is successful in line 140. Thus we don't have to call it twice and do the same operation a second time.

Reply:
Done